sunpy/net/tests/test_helioviewer.py

```diff
@@ -88,17 +89,20 @@ def test_get_jp2_header(self, client):
         assert ('fits' in header1.keys()) and ('fits' in header2.keys())
 
     @skip_glymur
+    @figure_test
     def test_download_jp2_map(self, client):
         """
-        Tests getJP2Image API method with Map.
+        Tests getJP2Image API method with Map with a figure test.
         """
         # TODO: make this a figure test.
         filepath = client.download_jp2('2012/01/01', observatory='SOHO',
                                        instrument='MDI', measurement='continuum')
```

Comment:
This isn't your doing, but this test should really be downloading to a `tempfile.TemporaryDirectory()`, with a call to `.cleanup()` at the end of the test.

Reply:
Alright, I shall make that change.